hash/ibm5170_cdrom.xml

```diff
@@ -3116,6 +3196,233 @@ https://www.pcgamingwiki.com/wiki/Myst
+		<year>1999</year>
+		<publisher>Red Orb Entertainment</publisher>
+		<notes><![CDATA[
+CD contents:
+Games:
+Prince of Persia v1.4
+Prince of Persia 2: The Shadow and the Flame v1.1
+Extras:
+The Making of Prince of Persia 3D
+Manual / User's Guide (PDF, 28 pages) (English)
+Demos:
+The Journeyman Project 3: Legacy of Time
+Riven: The Sequel to Myst
+Warlords III: Darklords Rising
+]]></notes>
+
+		<info name="language" value="English" />
+		<info name="region" value="Europe" />
+		<sharedfeat name="platform" value="DOS" />
+
+		<part name="cdrom" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="Prince of Persia Collection - Limited Edition (Europe)" sha1="878acef14a973ef7e988457bfec8bf46d13b7ffb" />
+			</diskarea>
+		</part>
+	</software>
+
+	<!-- Source: http://redump.org/disc/31619/ -->
+	<!-- <rom name="Prince of Persia Collection - Limited Edition (Europe) (Alt).cue" size="149" crc="12f45fc9" sha1="d4e3a8dd49c0f505a3a1b5453f55b849f92aa54c"/> -->
+	<!-- <rom name="Prince of Persia Collection - Limited Edition (Europe) (Alt).bin" size="731076864" crc="7aefb11e" sha1="bb84db27e5bceccc0041558f3a1157c77f9e8edd"/> -->
+	<software name="ppersialea" cloneof="ppersiale">
+		<description>Prince of Persia Collection: Limited Edition (Europe, alt)</description>
+		<year>1999</year>
+		<publisher>Red Orb Entertainment</publisher>
+		<notes><![CDATA[
+CD contents:
+Games:
+Prince of Persia v1.4
+Prince of Persia 2: The Shadow and the Flame v1.1
+Extras:
+The Making of Prince of Persia 3D
+Manual / User's Guide (PDF, 28 pages) (English)
+Demos:
+The Journeyman Project 3: Legacy of Time
+Riven: The Sequel to Myst
+Warlords III: Darklords Rising
+]]></notes>
+
+		<info name="language" value="English" />
+		<info name="region" value="Europe" />
+		<sharedfeat name="platform" value="DOS" />
+
+		<part name="cdrom" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="Prince of Persia Collection - Limited Edition (Europe) (Alt)" sha1="182794044998b3fe675c88e49e6fc5ae8da73c4d" />
+			</diskarea>
+		</part>
+	</software>
+
+	<!-- Source: http://redump.org/disc/9664/ -->
+	<!-- <rom name="Prince of Persia Collection - Limited Edition (USA).cue" size="117" crc="64ec00ff" sha1="e59aaab8e3da4384cc7d1b24035325b9f4aed1ed"/> -->
+	<!-- <rom name="Prince of Persia Collection - Limited Edition (USA).bin" size="723065952" crc="8030340e" sha1="1625c3d448f432afb06ff96b334278f2b930f942"/> -->
+	<software name="ppersiale_us" cloneof="ppersiale">
+		<description>Prince of Persia Collection: Limited Edition (USA)</description>
+		<year>1999</year>
+		<publisher>Red Orb Entertainment</publisher>
+		<notes><![CDATA[
+CD contents:
+Games:
+Prince of Persia (v1.4)
+Prince of Persia 2: The Shadow and the Flame (v1.1)
+Playable Demos:
+The Journeyman Project 3: Legacy of Time
+Riven: The Sequel to Myst
+Warlords III: Darklords Rising
+Videos:
+The Making of Prince of Persia 3D
+The Journeyman Project 3: Legacy of Time (included in demo)
```

Comment:
Does this mean there is also a video titled "The Journeyman Project 3: Legacy of Time" (same title as the game) on the CD?

Reply:
updated this information confirming on the CD (there is no video).